### PR TITLE
more aliases

### PR DESCRIPTION
**Pull request by Cashmaney**

### CODE REVIEW

File: packaging_docker/testnet/azure/secret-node-azure-template/scripts/secretnode.sh [hunk 1]
```diff
@@ -60,6 +60,14 @@ perl -i -pe 's/laddr = .+?26656"/laddr = "tcp:\/\/0.0.0.0:26656"/' ~/.secretd/co
 
 echo "Setting Secret Node environment variables" >> /home/$1/install.progress.txt
 
+echo 'alias secretcli="docker exec -it secret-node_node_1 secretcli"' >> /home/"$1"/.bashrc
+echo 'alias secretd="docker exec -it secret-node_node_1 secretd"' >> /home/"$1"/.bashrc
```
Comment (reuvenpo) [inline packaging_docker/testnet/azure/secret-node-azure-template/scripts/secretnode.sh:64]: wait, didn't i add these two before?

File: packaging_docker/testnet/azure/secret-node-azure-template/scripts/secretnode.sh [hunk 2]
```diff
@@ -60,6 +60,14 @@ perl -i -pe 's/laddr = .+?26656"/laddr = "tcp:\/\/0.0.0.0:26656"/' ~/.secretd/co
...
+echo 'alias show-node-id="docker exec -it bootstrap secretd tendermint show-node-id"' >> /home/"$1"/.bashrc
+echo 'alias show-validator="docker exec -it bootstrap secretd tendermint show-node-id"' >> /home/"$1"/.bashrc
```
Comment (reuvenpo) [inline packaging_docker/testnet/azure/secret-node-azure-template/scripts/secretnode.sh:66]: these two are the same. is this intentional?

File: packaging_docker/testnet/azure/secret-node-azure-template/scripts/secretnode.sh (hunk 1 above)
Comment (Cashmaney) [reply packaging_docker/testnet/azure/secret-node-azure-template/scripts/secretnode.sh:64]: To another branch

Comment (reuvenpo) [reply packaging_docker/testnet/azure/secret-node-azure-template/scripts/secretnode.sh:64]: ah, ok

Comment (reuvenpo) [reply packaging_docker/testnet/azure/secret-node-azure-template/scripts/secretnode.sh:64]: which one?

**Pull request merged**